_data/simple-icons.json

```diff
@@ -5227,6 +5227,11 @@
             "source": "https://en.wikipedia.org/wiki/File:Hyundai_Motor_Company_logo.svg",
             "guidelines": "https://www.hyundai.pl/fileadmin/user_upload/media/logo/201607_HYU_Guideline_ENG_small.pdf"
         },
+        {
+            "title": "Hyva Themes",
+            "hex": "0A23B9",
+            "source": "https://drive.google.com/file/d/1ohplW2WhEdAo8z5daUpFxvvtuJG56UsS/view"
```

Comment:
Perhaps better to use https://hyva.io/ https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines